libtsuba/include/tsuba/RDGInspection.h

```diff
@@ -0,0 +1,265 @@
+#ifndef KATANA_LIBTSUBA_RDGINSPECTION_H_
+#define KATANA_LIBTSUBA_RDGINSPECTION_H_
+
+#include <cstdlib>
+#include <iostream>
+#include <vector>
+
+#include "katana/BufferedGraph.h"
+#include "katana/FileGraph.h"
+#include "katana/Galois.h"
+#include "katana/LCGraph.h"
+#include "katana/OfflineGraph.h"
+#include "llvm/Support/CommandLine.h"
+
+namespace tsuba {
+
+namespace cll = llvm::cl;
+
+enum StatMode {
+  degreehist,
+  degrees,
+  maxDegreeNode,
+  dsthist,
+  indegreehist,
+  sortedlogoffsethist,
+  sparsityPattern,
+  summary
+};
+
+typedef katana::OfflineGraph Graph;
+typedef Graph::GraphNode GNode;
+
+using Writer = katana::FileGraphWriter;
+
+void
+doSummary(Graph& graph) {
+  std::cout << "NumNodes: " << graph.size() << "\n";
```

Comment:
use `fmt::print` everywhere and don't use `std::cout` and don't `#include <iostream>`